sql/src/main/java/io/crate/action/job/ContextPreparer.java

```diff
@@ -412,6 +412,24 @@ public ExecutionSubContext visitCollectPhase(final CollectPhase phase, final Pre
         }
 
         @Override
+        public ExecutionSubContext visitFileUriCollectPhase(FileUriCollectPhase phase, PreparerContext context) {
```

Comment:
looks like the same as RoutedCollectPhase visit method, why use a dedicated method?